### PR TITLE
Social Image and Meta tags

### PR DESCRIPTION
**Pull request by garrying**
- Follow up to https://github.com/ournetworks/2018/issues/27
- Adds OpenGraph and Twitter `<meta>`

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -4,7 +4,18 @@
     <meta charset="utf-8">
     <title>{% if page.title %}{{ page.title }}{% if page.subtitle %} {{ page.subtitle }}{% endif %} | {% endif %}{{ site.title }}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
+    {% capture sitedescription %}{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}{% endcapture %}
+    {% capture siteimage %}{{ "/images/social.png" | prepend: site.url }}{% endcapture %}
```
Comment (dcwalk) [inline _layouts/default.html]: do we want to add this to the config rather than do a capture?

File: _layouts/default.html [hunk 2]
```diff
@@ -4,7 +4,18 @@
     <meta charset="utf-8">
     <title>{% if page.title %}{{ page.title }}{% if page.subtitle %} {{ page.subtitle }}{% endif %} | {% endif %}{{ site.title }}</title>
```
Comment (dcwalk) [inline _layouts/default.html]: I think we need to revisit this with my switchup? Maybe strip/format the subtitle better

File: _layouts/default.html (hunk 1 above)
Comment (dcwalk) [reply _layouts/default.html]: (this = image path)

**Review by dcwalk — COMMENTED** [review]
looks good! Just a minor comment on whether using config make sense for image? I'm not sure but just imagining we are setting one sitewide, potentially with an override for yami-ichi?

**Comment by garrying** [comment]
Removed the subtitle var from the `title` for now. re Yami-Ichi: I think we could go with an override or have it be a part of a separate layout. I'm thinking the separate layout route would make sense if the layout is going to be way out from the current structure and styling 🤔

**Pull request merged**